### ISSUE TITLE
Random crashes involving ortp, mediastreamer_base and mediastreamer_voip

### ISSUE THREAD

**Issue by amaranth3** [BelledonneCommunications/linphone-iphone#420]
Hi,
We've been seeing a lot of random crashes involving ortp, mediastreamer_base and mediastreamer_voip. 
Here's the synopsis:
- LinphoneCore version: 3.11.2-161-ge336501
- Crashes happen randomly when initiating an outgoing call or while call is ongoing
- Crash logs attached
[mediastreamer and ortp crash logs.txt](https://github.com/BelledonneCommunications/linphone-iphone/files/1973078/mediastreamer.and.ortp.crash.logs.txt)

Do you know what possibly causes it and can you suggest a fix for this? Thanks

**Comment by ReisBenjamin** [comment]
Hello,

There's no symbol in your stack trace, it's unusable.
Please recompile your SDK with debug symbols (add -d to you prepare.py option)
and repost the stack with the symbols

**Comment by amaranth3** [comment]
I cloned a test project and re-ran liblinphone sdk update using the same command: ./prepare.py -c && ./prepare.py --debug && make.
The terminal output read: 
Install the project...
-- Install configuration: "Debug"

It seems to indicate that by issuing the above prepare.py update command, it does install the sdk with debug option enabled. This is the same command I ran previously in the current version of the sdk. Yet, the crash reports that point to problem with ortp, mediastreamer_base and mediastreamer_voip still do not appear to be symbolicated.
The crashes are happening a lot daily. Pls advice what else we should try in order to fix the problem. Thanks.
